plugins/generator-1.16.5/forge-1.16.5/templates/plant.java.ftl

```diff
@@ -325,7 +325,11 @@ import net.minecraft.util.SoundEvent;
 					<#if data.plantType == "growapable" || data.forceTicking>
 					.tickRandomly()
 					</#if>
+					<#if data.isSolid>
+					.notSolid().setOpaque((bs, br, bp) -> false)

```

Comment:
Did Mojang name this method wrong or does this combination make the plant solid?

Reply:
This is 1.16, so not Mojmaps

Reply:
So just a wrong name?

Reply:
Yes, pretty much